Source/WebCore/html/HTMLImageLoader.cpp

```diff
@@ -80,7 +81,7 @@ void HTMLImageLoader::notifyFinished(CachedResource&, const NetworkLoadMetrics&
     Ref<Element> protect(element());
     ImageLoader::notifyFinished(cachedImage, metrics);
 
-    bool loadError = cachedImage.errorOccurred() || cachedImage.response().httpStatusCode() >= 400;
+    bool loadError = cachedImage.errorOccurred();
```

Comment:
I don't think this code change actually causes any web facing behavior change? The chromium code change made more sense because loadError was used for various element types.

Here the only thing you changed is whether we report the extra memory to the GC or not it seems.